Core.DomainModel/ItContract/ItContract.cs

```diff
@@ -543,6 +543,8 @@ public void ClearMasterReference()
 
         #endregion
 
+        public virtual ICollection<DataProcessingRegistration> AssociatedDataProcessingRegistrations { get; set; }
+
         public virtual ICollection<SystemRelation> AssociatedSystemRelations { get; set; }
 
         public virtual ICollection<GDPR.DataProcessingRegistration> DataProcessingRegistrations { get; set; }
```

Comment:
when a DPR is removed from the ItContract, we should raise an event where the handler calls reset on dpr's which have the contract set as main contract. That ensures that the "modified" timestamp is updated on the DPR and not just that data has changed magiaclly behind the scenes because the database allowed it